Libraries/Geolocation/RCTLocationObserver.m

```diff
@@ -144,6 +144,14 @@ - (void)beginLocationUpdatesWithDesiredAccuracy:(CLLocationAccuracy)desiredAccur
   if ([[NSBundle mainBundle] objectForInfoDictionaryKey:@"NSLocationAlwaysUsageDescription"] &&
     [_locationManager respondsToSelector:@selector(requestAlwaysAuthorization)]) {
     [_locationManager requestAlwaysAuthorization];
+
+    // On iOS 9+ we also need to enable background updates
+    NSArray* backgroundModes  = [[NSBundle mainBundle] objectForInfoDictionaryKey:@"UIBackgroundModes"];
+    if(backgroundModes && [backgroundModes containsObject:@"location"]) {
+      if([_locationManager respondsToSelector:@selector(setAllowsBackgroundLocationUpdates:)]) {
+        [_locationManager setAllowsBackgroundLocationUpdates:YES];
+      }
+    }
```

Comment:
Will this always enable background updates when the app starts listening for location updates?

Or will it only be enabled when configured as described in `Geolocation.js`?

Sorry for the iOS n00b questions :)